concrete/themes/dashboard/elements/header.php

```diff
@@ -26,7 +27,7 @@
 if (!$frontendPageID) {
     $backLink = DIR_REL . '/';
 } else {
-    $backLink = DIR_REL . '/' . DISPATCHER_FILENAME . '?cID=' . $frontendPageID;
+    $backLink = Url::to(Page::getByID($frontendPageID));
```

Comment:
@JohntheFish Don't you need to import the `Url` class?

Reply:
I thought so too at first. 

As there are many Url classes, I searched the core for Url::to in order to discover which class to 'use' and found that many core files don't do so. Hence I tested without any import and it worked. 

I guess Url is already made available in some contexts - But I did need to import Page, which is strange because the Page class is often already available. It would be useful to have a simple docs page keeping track of the rules for what is and isn't already available.

Reply:
[`\URL` is an alias of the `\Concrete\Core\Support\Facade\Url` facade](https://github.com/concretecms/concretecms/blob/9.2.6/concrete/config/app.php#L183).

So, since this file is defined in the root namespace, in order to use `URL` we don't need to `use` anything.
But please remark that the correct spelling are:
- `URL`
- `Concrete\Core\Support\Facade\Url`

So, using `Url` is wrong: we should use `URL` or add `use Concrete\Core\Support\Facade\Url;`.

PHP works because class names are case-insensitive. But that's not a good reason to write class names with wrong letter casing.

I also prefer avoiding facades (and aliases)... Checking `Concrete\Core\Support\Facade\Url` we see that `URL::to(...)` is the same as calling `app('url/manager')->resolve([...])`.